config/routes.rb

```diff
@@ -1,11 +1,11 @@
 Rails.application.routes.draw do
   devise_for :users
   resources 'insurances', only: [:index, :show] do
-    resources :subscriptions, only: [:create]
+    resources :subscriptions, only: [:new, :create]
   end
 
   namespace :admin do
-    resources :insurances, only: [:new, :create]
+    resources :insurances, only: [:index, :new, :create, :edit, :update, :destroy, :show]
```

Comment:
We have all 7 of the standard actions, so no need to enumerate all of them 🙂 

```suggestion
    resources :insurances
```